### PR TITLE
Task 276 and 275 and 249 from enhanced wsc dataset

### PR DESCRIPTION
**Pull request by Palipoor**
This task is driven from the dataset released in paper "The Sensitivity of Language Models and Humans
to Winograd Schema Perturbations". These are perturbed Winograd Schema Challenge sentences and the task is like the WSC to find the given pronoun's reference.

### CODE REVIEW

File: tasks/README.md [hunk 1]
```diff
@@ -198,4 +198,5 @@ Name | Summary | Category
 `task205_remove_even_elements.json` | Given a list of integers remove all elements that are even | Answer Generation, Arithmetic
 `task206_collatz_conjecture.json` | Given a list of integers compute the next number in the *3n+1* problem | Answer Generation, Arithmetic
 `task207_max_element_lists.json` | Given a list of lists of integers compute the max value for each list | Answer Generation, Arithmetic
-`task208_combinations_of_list.json` | Given a list of integers of length *n* find all possible combinations,without replacement, of length *n-1* | Answer Generation, Combinatorics
\ No newline at end of file
+`task208_combinations_of_list.json` | Given a list of integers of length *n* find all possible combinations,without replacement, of length *n-1* | Answer Generation, Combinatorics
+`task249_enhanced_wsc_pronoun_disambiguation` | Given a sentence and a pronoun, decide which one of the choices is the pronoun refering to | Answer Generation, Pronoun Disambiguation
```
Comment (slowy07) [inline tasks/README.md]:
```suggestion
`task249_enhanced_wsc_pronoun_disambiguation` | Given a sentence and a pronoun, decide which one of the choices is the pronoun referring to | Answer Generation, Pronoun Disambiguation
```
it should ``referring`` not ``refering``

**Comment by Palipoor** [comment]
@slowy07 Fixed it!

**Review by slowy07 — APPROVED** [review]
LGTM!, thanks for the contribution

**Comment by danyaljj** [comment]
Looks reasonable to me too.

**Comment by Palipoor** [comment]
@slowy07 @danyaljj I just added another two tasks from the same dataset.

**Review by swarooprm — APPROVED** [review]
Looking good!
One comment: You can create more such tasks e.g. you can break the MCQ task to create incorrect answer generation and correct answer generation task. 
Overall, I like the diversity of the tasks you have added here.

**Pull request merged**